src/Symfony/Bundle/FrameworkBundle/Templating/Helper/RequestHelper.php

```diff
@@ -69,26 +60,24 @@ public function getLocale()
         return $this->getRequest()->getLocale();
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'request';
+    }
+
     private function getRequest()
     {
-        if ($this->requestStack) {
-            if (!$this->requestStack->getCurrentRequest()) {
+        if (null === $this->requestStack) {
```

Comment:
it cannot be null anymore (and your change is wrong btw, as you are changing the meaning of the condition, making broken)